Blogplace.Tests.Unit/Tests/PermissionsCheckerTests.cs

```diff
@@ -24,5 +19,33 @@ public Task PermissionsChecker_CanCreateArticle(CommonPermissionsEnum permission
                     
         return Task.CompletedTask;
     }
+
+    [Test]
+    [TestCase(CommonPermissionsEnum.ArticleRead, true)]
+    [TestCase(CommonPermissionsEnum.All & ~CommonPermissionsEnum.ArticleRead, false)]
+    public Task PermissionsChecker_CanReadArticle(CommonPermissionsEnum permissions, bool expected)
+    {
+        var permissionsChecker = new PermissionsChecker();
+
+        permissionsChecker
+            .CanReadArticle(permissions)
+            .Should().Be(expected);
+
+        return Task.CompletedTask;
+    }
+
+    [Test]
+    [TestCase(CommonPermissionsEnum.ArticleDelete, true)]
+    [TestCase(CommonPermissionsEnum.All & ~CommonPermissionsEnum.ArticleDelete, false)]
+    public Task PermissionsChecker_CanDeleteArticle(CommonPermissionsEnum permissions, bool expected)
```

Comment:
po co zwracasz taska jak nic asynchronicznego tu nie robisz? to może być zwykły void